components/nav/common.js

```diff
@@ -305,7 +306,8 @@ function LogoutObstacle ({ onClose }) {
               router.reload()
               return
             }
-
+            window.logoutInProgress = true
+            clearAuthCookies()
```

Comment:
we might not hit this line during logout because we get switched to another account, see code above

the code above is also prone to the race condition